src/modules/RootPart.rb

```diff
@@ -1589,6 +1589,19 @@ def MountVarIfRequired(fstab, root_device_current, manual_var_mount)
       MountVarPartition(var_partition_current)
     end
 
+    def has_pam_mount
+      # detect pam_mount encrypted homes
+      pam_mount_path = Installation.destdir + "/etc/security/pam_mount.conf.xml"
+      return false unless File.exists? pam_mount_path
+      Builtins.y2milestone("Detected pam_mount.conf, checking existence of ecrypted home dirs")
+      pam_mount_conf = SCR.Read(path(".anyxml"), pam_mount_path);
+      volumes = pam_mount_conf["pam_mount"][0]["volume"] || nil
```

Comment:
Does exist a chance that you get `nil` on any of those places? (I mean, pam_mount_conf["pam_mount"], etc.)?
In such a case, better something like:

```ruby
pam = pam_mount_conf.fetch("pam_mount", [])[0]
volumes = pam && pam["volume"]
```

If that's not the case, then just ignore this message.